File: c_practice_file/reverse_number.c
////////////// Using while loop ./////////

#include<stdio.h>
void main()
{
int n, remain,rev=0;
printf(" \n ************ reverse number **********\n\n");
printf("enter the number \t");
scanf("%d",&n); // input number 123

while(n!=0)
{
remain=n%10;  // 3
rev=(rev*10)+remain; //0*10+3=3 
n=n/10;  //12=123/10
}
printf("\n rev = %d\n",rev);

}

File: class_assignment/reverse_number.py
'''
Wap that accept a three digit number from user  and obtain
reverse of the number
'''

print("\nWap that accept a three digit number from user  and obtain reverse of the number\n")

num=int(input("Enter Three combination Digit \t"))
while num>999 or num<100:
    num=int(input("Enter Three combination Digit \t"))
    
rev=0
sum=0
while num!=0:
    remain=num%10
    rev=rev*10+remain
    sum+=remain
    num=num//10
print("Reverse Number of "," is - ",rev)
print("Sum of Reverse num is - ",sum)
